### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by luisdavim** [amouat/reggie#9]
Is this abandoned? will this functionality be integrated into https://github.com/ContainerSolutions/trow?

**Comment by amouat** [comment]
Yes, that's the plan. Unfortunately it's taking a bit longer than I hoped. Trow is a working registry now, but still needs a couple of basic features before I want to move on tackling the ImageWolf stuff (which I normally refer to as "advanced distribution").

I'm considering archiving this repo.
